xl-op/blueprint.yaml

```diff
@@ -227,6 +227,190 @@ spec:
+    - name: StorageClass
+      type: Select
+      options:
+        - !expr k8sResources(Namespace, 'storageclasses')
+      saveInXlvals: true
+      ignoreIfSkipped: true
+      overrideDefault: true
+      prompt: "Provide storage class for the server:"
+      promptIf: !expr ServerType != 'dai-release-runner' && ProcessType == "install"
+      description: Provide storage class for the server
+    - name: ResourcesSource
+      type: Select
+      options:
+        - label: Predefined resource values
+          value: preset
+        - label: Custom resource values
+          value: editor
+      saveInXlvals: true
+      ignoreIfSkipped: true
+      overrideDefault: true
+      prompt: "Select source of the resource values (CPU and memory):"
+      promptIf: !expr ServerType != 'dai-release-runner' && ProcessType == "install"
+      description: "Provide source of the resources (CPU and memory) requests and limits values for various containers. This selection will be used for specifying resources for all the installation components"
+      default: editor
+    - name: ResourcesPresetDeploy
+      type: Select
+      options:
+        - label: Request is 500m cpu and 1Gi memory. Limit is 1.0 cpu and 2Gi memory
+          value: nano
+        - label: Request is 1.0 cpu and 2Gi memory. Limit is 2.0 cpu and 4Gi memory
+          value: micro
+        - label: Request is 2.0 cpu and 5Gi memory. Limit is 4.0 cpu and 8Gi memory
+          value: small
+        - label: Request is 5.0 cpu and 10Gi memory. Limit is 8.0 cpu and 16Gi memory
+          value: medium
+        - label: Request is 10.0 cpu and 21Gi memory. Limit is 16.0 cpu and 32Gi memory
+          value: large
+        - label: Request is 21.0 cpu and 42Gi memory. Limit is 32.0 cpu and 64Gi memory
+          value: xlarge
+        - label: Request is 42.0 cpu and 85Gi memory. Limit is 64.0 cpu and 128Gi memory
+          value: 2xlarge                                                                     
+      saveInXlvals: true
+      ignoreIfSkipped: true
+      overrideDefault: true
+      prompt: "Select one of the predefined resource values for Deploy master:"
+      promptIf: !expr ServerType == 'dai-deploy' && ProcessType == "install" && ResourcesSource == "preset"
+      description: "Select one of the predefined resource values for Deploy master"
+      default: nano         
+    - name: ResourcesEditorDeploy
+      type: Editor
+      saveInXlvals: true
+      ignoreIfSkipped: true
+      overrideDefault: true
+      prompt: "Provide resource allocation for Deploy master:"
+      promptIf: !expr ServerType == 'dai-deploy' && ProcessType == "install" && ResourcesSource == "editor"
+      description: "Provide resource allocation for Deploy master"
+      default: |-
```

Comment:
Create the anchor for resources editor default template so it can be referenced throughout the file